newIDE/app/src/EventsSheet/ParameterFields/KeyField.js

```diff
@@ -95,12 +105,9 @@ const keyNames = [
   'F11',
   'F12',
   'Pause',
-  'RControl',
-  'LControl',
-  'RAlt',
-  'LAlt',
-  'RShift',
-  'LShift',
+  'LBracket', // seems not working
+  'RBracket', // seems not working
+  'SemiColon', // seems not working
```

Comment:
Semicolon seems working according to https://keycode.info/ (keycode 186) (notice the lowercase c). Might be something we need to add in the game engine?